.github/workflows/deploy.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: ci
```

Comment:
I'm pretty sure @macpijan had some generic pre-commit integration project.

Reply:
@pietrushnic pre-commit and SEO yml will be added later, I didn't want to add everything on initial deploy